CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 * [ENHANCEMENT] Experimental TSDB: Add support for local `filesystem` backend. #2245
 * [ENHANCEMENT] Allow 1w (where w denotes week) and 1y (where y denotes year) when setting table period and retention. #2252 
 * [ENHANCEMENT] Added FIFO cache metrics for current number of entries and memory usage. #2270
+* [ENHANCEMENT] Remove omitempty from structs. #2209
```

Comment:
Try to phrase this from the point of view of someone who deploys Cortex rather than someone who works on the code.
E.g. "Output all config fields to `/config` API, even those with zero value"

Reply:
👍